pynest/nest/lib/hl_api_simulation.py

```diff
@@ -121,7 +130,11 @@ def Prepare():
 
 @check_stack
 def Cleanup():
-    """Cleans up resources after a `Run` call. Not needed for `Simulate`.
+    """Cleans up resources after a `Run` call.
+
+    `Cleanup` is automatically triggered at the end of call to `Simulate`:
+
+    Simulate(t): t' = t/m; Prepare(); for _ in range(m): Run(t'); Cleanup()
```

Comment:
```suggestion
    """Cleans up resources after a `Run` calls.

    `Cleanup` is automatically called by `Simulate` and `RunManager` .
```

Also add `Simulate, RunManager` to See Also below.